### PR TITLE
Update sideload-office-add-ins-for-testing.md

### PR DESCRIPTION
**Pull request by EsterBergen**
Updated the "Manually sideload an addin section" - step 2 said to go to the insert tab to select addins when addins are actually in the home tab.
 I updated "insert tab" to "home tab"

### CODE REVIEW

**Comment by AlexJerabek** [comment]
@EsterBergen, Thanks for letting me hijack this PR to make a bunch of changes across the board. @samantharamon, please direct your feedback to me.

**Comment by AlexJerabek** [comment]
@samantharamon, the old images have been removed and a new CF screenshot has been added.

File: docs/testing/debug-add-ins-using-f12-tools-ie.md [hunk 1]
```diff
@@ -63,7 +63,7 @@ The following steps are the instructions for debugging your add-in. If you just
 This example uses Word and a free add-in from AppSource.
 
 1. Open Word and choose a blank document.
-1. On the **Insert** tab, in the **Add-ins** group, select **My Add-ins** to open the **Office Add-ins** dialog and then select the **STORE** tab.
+1. Select **Home** > **Add-ins**, then select **Get Add-ins**. to open the **Office Add-ins** dialog and then select the **STORE** tab.
```
Comment (samantharamon) [inline docs/testing/debug-add-ins-using-f12-tools-ie.md:66]:
```suggestion
1. Select **Home** > **Add-ins**, then select **Get Add-ins**.
1. In the **Office Add-ins** dialog, select the **STORE** tab.
```

File: docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md [hunk 2]
```diff
@@ -86,7 +86,7 @@ For Office applications on the desktop, you can find Office Add-ins from the app
     Specify just the URL of the parent site collection:
     - `https://<domain>/sites/<AddinCatalogSiteCollection>`
 1. Close and reopen the Office application.
-1. Choose **Insert** > **Get Add-ins**.
+1. Select **Home** > **Add-ins**, then select **Get Add-ins**.
 1. In the Office Add-ins dialog, choose the **MY ORGANIZATION** tab.
```
Comment (samantharamon) [inline docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md:90]:
```suggestion
1. In the Office Add-ins dialog, choose the **ADMIN MANAGED** tab.
```

File: docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ For online Office applications, you can find Office Add-ins from the app catalog
 
 1. Open the online Office application (Excel, PowerPoint, or Word).
 1. Create or open a document.
-1. Choose **Insert** > **Add-ins**.
+1. Select **Home** > **Add-ins**, then select **Get Add-ins**.
 1. In the Office Add-ins dialog, choose the **MY ORGANIZATION** tab.
```
Comment (samantharamon) [inline docs/publish/publish-task-pane-and-content-add-ins-to-an-add-in-catalog.md:74]:
```suggestion
1. In the Office Add-ins dialog, choose the **ADMIN MANAGED** tab.

```

File: docs/overview/office-add-ins.md [hunk 4]
```diff
@@ -85,7 +85,7 @@ You can add custom ribbon buttons and menu items to the ribbon in Office on the
 
 #### Task panes  
 
-You can use task panes in addition to add-in commands to enable users to interact with your solution. Clients that don't support add-in commands (Office on iPad) run your add-in as a task pane. Users launch task pane add-ins via the **My Add-ins** button on the **Insert** tab.
+You can use task panes in addition to add-in commands to enable users to interact with your solution. Clients that don't support add-in commands (Office on iPad) run your add-in as a task pane. Users launch task pane add-ins via the **Home** > **Add-ins** button.
```
Comment (samantharamon) [inline docs/overview/office-add-ins.md:88]:
```suggestion
You can use task panes in addition to add-in commands to enable users to interact with your solution. Clients that don't support add-in commands (Office on iPad) run your add-in as a task pane. In Excel, Word, and PowerPoint, users launch task pane add-ins via the **Home** > **Add-ins** button. In Outlook, users launch task pane add-ins via the add-in button or via the **All Apps** button on the ribbon.
```

File: docs/includes/excel-custom-functions-manually-register.md [hunk 5]
```diff
@@ -1,22 +1,18 @@
 If the `CONTOSO` namespace isn't available in the autocomplete menu, take the following steps to register the add-in in Excel.
 
-### [Excel on Windows or Mac](#tab/excel-windows)
+# [Excel on Windows or Mac](#tab/excel-windows)
 
-1. In Excel, choose the **Insert** tab and then choose the down-arrow located to the right of **My Add-ins**.
+1. In the Excel ribbon, select the **Home** > **Add-ins**.
```
Comment (samantharamon) [inline docs/includes/excel-custom-functions-manually-register.md:5]:
```suggestion
1. In the Excel ribbon, select **Home** > **Add-ins**.
```

File: docs/includes/excel-custom-functions-manually-register.md [hunk 6]
```diff
@@ -1,22 +1,18 @@
...
-
-    :::image type="content" source="../images/excel-cf-tutorial-register.png" alt-text="Screenshot of the Insert ribbon in Excel on Windows, with the Excel Custom Functions add-in highlighted in the My Add-ins list.":::
+    :::image type="content" source="../images/excel-cf-select-add-in.png" alt-text="The My Add-ins dialog that shows active add-ins, with the My custom function add-in button highlighted.":::
 
 # [Excel on the web](#tab/excel-online)
 
-1. In Excel, choose the **Insert** tab and then choose **Add-ins**.
-
-    :::image type="content" source="../images/excel-cf-online-register-add-in-1.png" alt-text="Screenshot of the Insert ribbon in Excel on the web, with the My Add-ins button highlighted.":::
+1. In Excel, select **Home** > **Add-ins**, then select **Get Add-ins**.
```
Comment (samantharamon) [inline docs/includes/excel-custom-functions-manually-register.md:13]:
```suggestion
1. In Excel, select **Home** > **Add-ins**, then select **More Add-ins**.
```

File: docs/includes/excel-custom-functions-manually-register.md [hunk 7]
```diff
@@ -1,22 +1,18 @@
...
 
-1. Choose **Manage My Add-ins** and select **Upload My Add-in**.
+1. Under the **MY ADD-INS**, select **Manage My Add-ins** and choose **Upload My Add-in**.
```
Comment (samantharamon) [inline docs/includes/excel-custom-functions-manually-register.md:15]:
```suggestion
1. In the **MY ADD-INS** tab, select **Manage My Add-ins** and choose **Upload My Add-in**.
```

**Review by samantharamon — APPROVED** [review]

**Review by samantharamon — COMMENTED** [review]
Thanks for making these changes. Approving with suggestions for your review.

File: docs/includes/excel-custom-functions-manually-register.md (hunk 6 above)
Comment (AlexJerabek) [reply docs/includes/excel-custom-functions-manually-register.md:13]: image

Unless I'm mistaken, the "More" button doesn't always show up.

Comment (samantharamon) [reply docs/includes/excel-custom-functions-manually-register.md:13]: I consistently get the "Get Add-ins" option in the WXP desktop clients, and the "More Add-ins" in the web clients.

Comment (AlexJerabek) [reply docs/includes/excel-custom-functions-manually-register.md:13]: Ah, you're right... What fun!

**Pull request merged**